src/redux/actions/projects/loadProjects.js

```diff
@@ -53,7 +53,7 @@ const loadProjects = () => async (dispatch) => {
 
       await Promise.all(data
         .filter((entry) => entry.samples.length)
-        .map((entry) => dispatch(loadSamples(false, entry.uuid))));
+        .map((entry) => dispatch(loadSamples(null, entry.uuid))));
```

Comment:
why passing null here?

Reply:
this was a minor fix, this line was using `false` in the same way as `null` relying on the fact that we check for nulls by doing `!experimentId`. `null` is way clearer about the fact that this is not being deinfed on purpose and doesn't vary the type of what we are passing between `string` and `bool`.